students/eowyn/windrevenue/windrevenue/power_curve_tool.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+""" 
+Power curves for Backcast
+
+Keep track of previously used power curves as .json files
+Print a list of available power curves
+Read new power curves
+Read and write power curves from/to .json files
+"""
+
+import pandas as pd
+import numpy as np
+
+class power_curve_tool():
```

Comment:
Without the content, I can't tell, but is there a good reason to put these in classes? If they are essentially stand alone functions, they can simple be functions in a module.